client/src/components/Comics.js

```diff
@@ -38,15 +38,30 @@ function Comics() {
     setCurrentPage(1);
     setSearchString(term);
   };
+  const comicsContainer = {
```

Comment:
It would probably be best to move these to a module.scss file and use `className` so we're being consistent throughout the project